### PR TITLE
Consolidating the helper apks

### PR DESCRIPTION
**Pull request by prakashcodes25**
This commit includes the consolidation of both [io.appium.settings](https://github.com/appium/io.appium.settings) and [unlock_apk](https://github.com/appium/unlock_apk) apks.

### CODE REVIEW

**Comment by prakashcodes25** [comment]
[appium#6089](https://github.com/appium/appium/issues/6089)

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.2"
+
+    defaultConfig {
+        applicationId "io.appium.helper"
+        minSdkVersion 17
```
Comment (bootstraponline) [inline app/build.gradle]: appium supports selendroid which means we need to match their min sdk. Run time SDK checks are used to verify we're not calling unsupported methods.

minSdk should be [10](https://github.com/selendroid/selendroid/blob/04550f8d20bd668120a81a5e43f068154d4e34d8/selendroid-server/AndroidManifest.xml#L4)

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Wed Oct 21 11:34:03 PDT 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-all.zip
```
Comment (bootstraponline) [inline gradle/wrapper/gradle-wrapper.properties]: should be gradle 2.11

**Comment by bootstraponline** [comment]
FYI when you update a pull request with a force push, there's no notification to anyone that the PR has changed. It's best to always comment after pushing a change so github sends an email notification.

**Comment by bootstraponline** [comment]
Code looks good to me. Merged.

**Pull request merged**